recipes/triqs_tprf/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+xorg-x11-server-Xorg
```

Comment:
Why do you need this?  Nothing is depending on X11 here

Reply:
I based this recipe our [triqs library feedstock](https://github.com/conda-forge/triqs-feedstock)
where this file was added an automatic [PR](https://github.com/conda-forge/triqs-feedstock/pull/3) related to our matplotlib-base dependency. I guess I can remove this file here?

Reply:
Yeah, unless it affects the testing somehow

Reply:
Removed it